execution/functions/functions.go

```diff
@@ -1156,6 +1156,28 @@ var FuncDuration = execution.Function{
 	},
 }
 
+var FuncImpute = execution.Function{
```

Comment:
A similar function to this one is called either IsNull or Coalesce in SQL naming - maybe either of these names is more appropriate? (I think there was also one supported only bo PostgreSQL, but can't remember the name of it now)

Reply:
You're right, I'll use coalesce and add ifnull